packages/test-commerce-sdk-react/app/pages/use-shopper-login-helper.tsx

```diff
@@ -18,7 +18,7 @@ const UseShopperLoginHelper = () => {
         <>
             <h1>LoginGuestUser</h1>
             <Json data={loginGuestUser} />
-            <button onClick={() => loginGuestUser.execute()}>loginGuestUser</button>
+            <button onClick={() => loginGuestUser.mutate()}>loginGuestUser</button>
```

Comment:
I saw an Typescript error 
![image](https://user-images.githubusercontent.com/52219283/194183329-1eb07687-db37-4e43-a6c9-d12ba4eb3d5d.png)

Reply:
the error should be gone now (dont' forget to rebuild)